servers/express.js

```diff
@@ -118,6 +207,8 @@ server.start()
     // eslint-disable-next-line no-console
     console.log(process.pid);
   })
-  .then(async () => {
-    await collector();
+  .then(async() => {
+    if (pathToLogFile) {
+      await collector();
```

Comment:
Keep in mind that for that case that works, but imagine this is a server that would be used by multiple users, then having a "global variable"(not a "request scope") could be a problem.